From 9ff504cd2aeeab117a629e57c6d34a157c4e8405 Mon Sep 17 00:00:00 2001
From: Dave Airlie <airlied@redhat.com>
Date: Tue, 12 Apr 2016 09:54:02 +1000
Subject: [PATCH 0547/1722] drm/amd/dal: drop unsued struct members.

Not sure why these aren't used, seems like they should be somewhere.

Signed-off-by: Dave Airlie <airlied@redhat.com>
Acked-by: Harry Wentland <harry.wentland@amd.com>
Signed-off-by: Alex Deucher <alexander.deucher@amd.com>
Signed-off-by: Kalyan Alle <kalyan.alle@amd.com>
---
 drivers/gpu/drm/amd/dal/include/hw_sequencer_types.h | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/drivers/gpu/drm/amd/dal/include/hw_sequencer_types.h b/drivers/gpu/drm/amd/dal/include/hw_sequencer_types.h
index 6af0d1d..9a78097 100644
--- a/drivers/gpu/drm/amd/dal/include/hw_sequencer_types.h
+++ b/drivers/gpu/drm/amd/dal/include/hw_sequencer_types.h
@@ -59,9 +59,6 @@ struct hw_crtc_timing {
 	/* in KHz */
 	uint32_t pixel_clock;
 
-	enum dc_color_depth color_depth;
-	enum dc_pixel_encoding pixel_encoding;
-
 	struct {
 		uint32_t INTERLACED:1;
 		uint32_t DOUBLESCAN:1;
-- 
2.7.4

